vignettes_overview.qmd

```diff
@@ -0,0 +1,24 @@
+---
+title: "Vignettes Overview"
+editor: visual
+---
+
+## Purpose
+
+The vignettes are to document technical skills and approaches that our group members learn, that are not well-documented in other places, e.g., online tutorials, blog posts.
+Creating these vignettes provide a way to share knowledge amongst and outside the group as we learn new things.
+
+## Process
+
+1.  Investigate the desired skill using existing documentation and by using the tool / approach
+2.  Summarize findings in a Quarto document and add it to this section of the Group Procedures book
```

Comment:
Could also just be plain .md—Quarto will render those as well.  Another (sub) step here is to add the file path to `_quarto.yml`